test/test_w3c_spec/test_trix_w3c.py

```diff
@@ -0,0 +1,129 @@
+"""This runs the nt tests for the W3C RDF Working Group's N-Quads
+test suite."""
```

Comment:
```suggestion
```

Reply:
I changed it to:
`"""This runs the TriX tests for RDFLib's informally-assembled TriX
test suite."""
`
might as well be explicit.